src/ui/assets/AssetsList.js

```diff
@@ -130,8 +130,9 @@ class AssetsList extends Component {
   }
 
   removeAllActiveFilters() {
-    const { onRemoveAllActiveFilters } = this.props;
+    const { onRemoveAllActiveFilters, onResetFilterinCategories } = this.props;
     onRemoveAllActiveFilters();
+    onResetFilterinCategories();
```

Comment:
It seems that there's a minor typo.
```suggestion
    onResetFilteringCategories();
```